### DISCUSSION TITLE
User A ID not change but Facing Issue Error - ID Does Not Exist.

### DISCUSSION THREAD

**Discussion by Shubhamjpg** [rustdesk/rustdesk#2870] [General]
![image](https://user-images.githubusercontent.com/27909412/209808073-3da5b76d-6443-4e00-9ffc-cae76d40f51f.png)
![image](https://user-images.githubusercontent.com/27909412/209808225-496c44e1-a227-42a3-ba66-db3beb7025d7.png)

**Comment by rustdesk** [comment]
We are migrating servers.

**Comment by Shubhamjpg** [comment]
This is not enhancement this is a bug It is my mistake a not abe to edit this is critical bug

**Comment by rustdesk** [comment]
try later.

**Comment by Shubhamjpg** [comment]
Thanks for informing May I hope your Migration should be Smoothly.

**Comment by phyzjqkl** [comment]
Dear rustdesk

4 day passed, the error still exist for my ID. Any solution?

**Comment by rustdesk** [comment]
We still have migration issue, https://twitter.com/rustdesk/status/1608770211968344065. We are still migrating to new servers.

**Comment by rustdesk** [comment]
> exist for my ID

That's because the remote side and local side connect to different servers (one is new, the other one is old), though the dns is changed, but your computer may not change, you may need to update your dns cache on your computer.

**Comment by rustdesk** [comment]
self-hosting is your best choice, the public servers are for demo only, not recommendated  for production.

**Comment by grummbeer** [comment]
@Shubhamjpg @phyzjqkl solved? can this be closed?
